test-app/.eslintrc.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+module.exports = {
+  root: true,
+  extends: ['eslint:recommended', 'plugin:prettier/recommended'],
+  plugins: ['prettier'],
+  parser: '@babel/eslint-parser',
+  parserOptions: {
+    ecmaVersion: 2017,
+    sourceType: 'module',
+    requireConfigFile: false, // allow @babel/eslint-parser to run on files that do not have a Babel configuration associated with them
+  },
+  env: {
+    browser: true,
+  },
+  rules: {},
+  overrides: [
+    // node files
+    {
+      files: [
+        '.eslintrc.js',
+        '.prettierrc.js',
+        '.template-lintrc.js',
+        'ember-cli-build.js',
+        'index.js',
+        'testem.js',
+        'blueprints/*/index.js',
+        'config/**/*.js',
+        'tests/dummy/config/**/*.js',
+      ],
+      excludedFiles: [
+        'addon/**',
+        'addon-test-support/**',
+        'tests/dummy/app/**',
+        'vendor/**',
+      ],
+      parserOptions: {
+        sourceType: 'script',
+      },
+      env: {
+        browser: false,
+        node: true,
+      },
+      plugins: ['node'],
```

Comment:
**suggestion**

Before updating the configuration files, replace `eslint-plugin-node` with `eslint-plugin-n`.

Related comment: https://github.com/emberjs/ember-qunit/pull/1062/commits/4d211f725911155f1a4a2d5621528e398e81c6cd#r1184576545

Reply:
This is done in another commit